render-ws/src/main/scripts/docker-compose/docker-compose.swarm.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.9" # see https://docs.docker.com/compose/compose-file/compose-file-v3
+services:
+  render-ws:
+    image: ${RENDER_WS_IMAGE}
+    user: ${JETTY_RUN_AS_USER_AND_GROUP_IDS}
+    ports:
+      - "8080:8080"
+
+    # Specify configuration related to the deployment and running of services.
+    deploy:
+
+      # One instance of the service on every node in the swarm.
+      mode: global
+
+      # Docker assigns the service a virtual IP (VIP) that acts as the front end for clients to reach the service on a network.
+      # Docker ingress routing mesh routes requests between the client and available worker nodes for the service,
```

Comment:
It took me a while to parse that sentence, because I wasn't familiar with the term "Docker ingress routing mesh". Is there a way of making this comment clearer, or is it clear to people familiar with Docker, anyway?